web/src/components/FeedbackModal.tsx

```diff
@@ -27,7 +27,7 @@ const ModalContainer = styled.div`
   right: 0;
   bottom: 0;
   left: 0;
-  z-index: 11;
+  z-index: 111;
```

Comment:
Modal should always be on top. in this case over the `BottomActionSheet`

Reply:
111 is not a good value though.

If it needs to be bigger than 110, use 120, so we still have the possibility to put a element between, if needed in the future. If 100 works, use this.